shacl12-core/index.html

```diff
@@ -701,35 +859,54 @@ <h3>SHACL Example</h3>
 	sh:closed true ;
 	sh:ignoredProperties ( rdf:type ) .
 						</div>
-					</div>
-				</aside>
-				<p>
-					The example below shows the same shape definition as a possible JSON-LD [[json-ld]] fragment.
-					Note that we have left out a <code>@context</code> declaration, and depending on the
-					<code>@context</code> the rendering may look quite different.
-					Therefore this example should be understood as an illustration only.
-				</p>
-				<pre class="jsonld">
-{
-	"@id" : "ex:PersonShape",
-	"@type" : "NodeShape",
-	"targetClass" : "ex:Person",
-	"property" : [
+						<div class="jsonld">
+							<pre class="jsonld">{
+	"@id": "ex:PersonShape",
+	"@type": "sh:NodeShape",
+	"sh:closed": {
+		"@type": "xsd:boolean",
+		"@value": "true"
+	},
+	"sh:ignoredProperties": {
+		"@list": [
+			{
+				"@id": "rdf:type"
+			}
+		]
+	},
+	"sh:property": [
 		{
-			"path" : "ex:ssn",
-			"maxCount" : 1,
-			"datatype" : "xsd:string" ,
-			"pattern" : "^\\d{3}-\\d{2}-\\d{4}$"
+			"sh:datatype": {
+				"@id": "xsd:string"
+			},
+			"sh:maxCount": {
```

Comment:
Change from using JSON native type to inline RDF literal - see comment about `@context` above.